spicetify.go

```diff
@@ -115,16 +114,15 @@ func init() {
 	}
 
 	if isAdmin.Check(bypassAdminCheck) {
-		utils.PrintError("Spicetify should not be run with administrator/root privileges")
-		utils.PrintError("Running as admin can cause Spotify to show a black/blank window after applying spicetify")
+		utils.PrintError("Spicetify should NOT be ran with administrator/root privileges")
```

Comment:
English modals - verb is in bare infinitive

```suggestion
		utils.PrintError("Spicetify should NOT be run with administrator or root privileges")
```